ccdproc/conftest.py

```diff
@@ -7,16 +7,29 @@
 import os
 
 try:
-    from astropy.tests.plugins.display import (pytest_report_header,
-                                               PYTEST_HEADER_MODULES,
+    # When the pytest_astropy_header package is installed
+    from pytest_astropy_header.display import (PYTEST_HEADER_MODULES,
                                                TESTED_VERSIONS)
+
+    def pytest_configure(config):
+        config.option.astropy_header = True
 except ImportError:
```

Comment:
I think we should just add `pytest-astropy-header` as a hard dependency  rather than doing this nested try except (it should ideally be picked up from pytest-astropy, but clearly something is not working as it should).

Reply:
> I think we should just add pytest-astropy-header as a hard dependency

Yeah, I also thought about this. But the test-log shows that it's in fact installed ([corresponding log line from your PR](https://travis-ci.org/astropy/ccdproc/jobs/614284643#L964)) but not picked up makes it probably a lot harder to solve than just adding a hard-dependency.

I would prefer the nested try-catch for now, it's only astropy-2.x after all and we may even drop support for it once the new astropy LTS is out.